### PR TITLE
Finish comments and update admin feature

### PR DESCRIPTION
**Pull request by nguyenhuynhquang2909**

### CODE REVIEW

File: server/components/models/userModel.js
Comment (KhangTGR) [inline server/components/models/userModel.js:1]: deault????

Comment (KhangTGR) [inline server/components/models/userModel.js:1]: deault???

**Review by KhangTGR — CHANGES_REQUESTED** [review]
change userModel.js

File: server/components/models/userModel.js
Comment (JustAbusiness) [reply server/components/models/userModel.js:1]: new english, bro

**Review by KhangTGR — APPROVED** [review]
niceeee

**Pull request merged**